### PR TITLE
Merge 5.10 into 5.11

### PR DESCRIPTION
**Pull request by mrunge**
This was missed before branching 5.11.

### CODE REVIEW

**Comment by rubenk** [comment]
Indeed, this is not the same result I'm getting. Please redo this.

**Comment by rubenk** [comment]
The result of my merge attempt is in #3408

File: .travis.yml [hunk 1]
```diff
@@ -38,9 +38,9 @@ script:
   - if [[ "${TRAVIS_BRANCH}" == "coverity_scan" ]]; then exit 0; fi
   - type pkg-config
   - pkg-config --list-all | sort -u
-  - ./configure
+  - ./configure --disable-lvm
```
Comment (rpv-tomsk) [inline .travis.yml]: This plugin was completely removed in this branch.

**Comment by mrunge** [comment]
@rubenk patch looks better. Closing this.

**Pull request closed without merging**